### PR TITLE
Modular coolant setup for TACOCAT

### PR DESCRIPTION
**Pull request by sierratutwiler**
Creating modular coolant capabilities for TACOCAT

- calls different coolant modules

- user selects coolant from predefined list of coolants

### CODE REVIEW

File: TACOCAT.py [hunk 1]
```diff
@@ -47,15 +46,43 @@
...
-rhoK = NaK_Prop.rhoK(Tbulkin + 273.15)
-CpNa = NaK_Prop.CpNa(Tbulkin + 273.15)
-CpK = NaK_Prop.CpK(Tbulkin + 273.15)
-Cp = (10**3)*(0.22*CpNa + 0.78*CpK) #J/kg-K
-k = NaK_Prop.k(Tbulkin + 273.15)
-nu = NaK_Prop.nu(Tbulkin + 273.15)
-invrhoNaK = 0.22/rhoNa + 0.78/rhoK
-rho = 1/invrhoNaK #kg/m^3
+CoolantUsed = int(input('Enter the number for the coolant you would like to use: 1. NaK 2. FLiBe 3. FLiNak 4. NaF-ZrF4:  '))
+#Coolants = {'1' : 'NaK_Prop','2' : 'FLiBe_Prop'}
```
Comment (lcarasik) [inline TACOCAT.py]: Do you need this line still?

**Review by lcarasik — CHANGES_REQUESTED** [review]
Check out the one comment and then fix it, let me know

File: TACOCAT.py (hunk 1 above)
Comment (sierratutwiler) [reply TACOCAT.py]: I think I fixed the issue, can you check?

**Review by lcarasik — APPROVED** [review]

**Comment by lcarasik** [comment]
All changes are fixed, we do want to look into dictionaries still.

https://www.w3schools.com/python/python_dictionaries.asp

**Pull request merged**